docs/rfc/002_standard_memo_format.md

```diff
@@ -0,0 +1,220 @@
+# A proposed standard memo format for non-EVM chains
+<br>
+
+## The goal
```

Comment:
Unrelated to this PR specifically, but I think it's useful to have a "Status: {proposed, declined, implemented}" on the RFCs.

In this case it's "proposed". In the PR actually implementing/completing the feature, you would set it to "implemented" and add a small description of any changes needed during the implementation.

Even declined RFCs should still be committed to the repo for historical reference.